frontend/src/components/suggestions.module.css

```diff
@@ -1,20 +1,106 @@
+.container-wrapper {
+  position: relative;
+  display: flex;
+  max-width: 600px;
+  width: 100%;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+.contentWrapper {
+  position: relative;
+  display: flex;
+  width: 100%;
+  overflow: hidden;
+}
+
 .container {
+  display: grid;
+  grid-auto-flow: column;
+  gap: 16px;
+  overflow-x: auto;
+  position: relative;
   align-items: stretch;
-  display: flex;
-  flex-direction: row;
   width: 100%;
-  justify-content: space-evenly;
+  padding: 0px 2px 8px 2px;
+}
+
+.container > div {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: var(--primary);
+  border-radius: 16px;
+  padding: 0;
+  width: 300px;
+  white-space: normal;
+  position: relative;
+}
+
+.container > div > p {
```

Comment:
I know this is just extending the selectors already here, but it would be good to switch these to simple class based selectors if possible?